pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/KitUploadRoute.java

```diff
@@ -234,7 +234,7 @@ private void uploadKit(@NonNull DDPInstance ddpInstance, @NonNull KitType kitTyp
                 String collaboratorParticipantId = "";
                 //if kit has ddpParticipantId use that (RGP!) and
                 //if ddpInstance is 30 then it is Darwin Vertebrate so we ignore this check
-                if (!ddpInstance.getDdpInstanceId().equals("30") && StringUtils.isBlank(kit.getParticipantId())) {
+                if(ddpInstance.getParticipantIndexES() != null && StringUtils.isBlank(kit.getParticipantId())) {
```

Comment:
NIT: please change to `StringUtils.isNotBlank(ddpInstance.getParticipantIndexES())`